frontend/src/Components/MyTasks/DeliveriesContainer.tsx

```diff
@@ -150,7 +151,7 @@ const DeliveriesContainer = (props: {
 
   // filtering logic
   const dateFilteredTasks = dummyTasks.filter(
```

Comment:
is the history page still using dummyTasks? since this PR is for connecting to backend, there shouldn't be any dummyTasks and it should display whatever is in the db

Reply:
but this is deliveries. History is using fetched tasks

Reply:
nvm it uses tasks now